front/lib/api/assistant/visualization.ts

```diff
@@ -119,23 +119,19 @@ The generated component should not have any required props / parameters.
 
    
 
-### Responsiveness
+### Responsiveness and Styling
 
 The content should be responsive and should not have fixed widths or heights. The component should be able to adapt to different screen sizes.
 The content should never overflow the viewport and should never have horizontal or vertical scrollbars.
 
 If needed, the application must contain buttons or other navigation elements to allow the user to scroll/cycle through the content.
-Never use tailwind's specific values like \`h-[600px]\`.
+Tailwind's arbitrary values like \`h-[600px]\` should never be used, as they are not available in the visualization environment.
```

Comment:
🤔

Reply:
Oups

Reply:
(this is already in prod, just didn't rebase this branch)